ruby/rubocop_lint.yml

```diff
@@ -111,16 +111,6 @@ Lint/IneffectiveAccessModifier:
                  the visibility of a class method, which does not work.
   Enabled: true
 
-Lint/InvalidCharacterLiteral:
```

Comment:
what happened to these two?

Reply:
looks like it wasn't really working...

https://github.com/bbatsov/rubocop/pull/4086/commits